app/admin/main_goals.rb

```diff
@@ -0,0 +1,59 @@
+ActiveAdmin.register MainGoal do
+  permit_params :name,
+      :spanish_name,
+      :id,
+      :created_at,
+      :updated_at,
+      :displayable,
+      :for_volunteer,
+      :for_client
```

Comment:
If users have a MainGoal association, then we can sort the users table by main goal as well. Why do we want to have this as an additional admin page?